src/main/java/com/minecolonies/coremod/commands/BackupCommand.java

```diff
@@ -15,10 +15,9 @@
 /**
  * Created by asie on 2/16/17.
  */
-public class BackupCommand extends AbstractSingleCommand
-{
-    public static final String DESC = "backup";
-    public static final String NO_PERMISSION_MESSAGE = "You do not have permission to backup colony data!";
+public class BackupCommand extends AbstractSingleCommand {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)